pkg/get/tools.go

```diff
@@ -1609,5 +1609,32 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 		},
 	)
 
+	tools = append(tools,
+		Tool{
+			Owner:       "wagoodman",
+			Repo:        "dive",
+			Name:        "dive",
+			Version:     "0.10.0",
+			Description: "A tool for exploring each layer in a docker image",
+			URLTemplate: `{{$osStr := ""}}
+			{{- if HasPrefix .OS "ming" -}}
+			{{$osStr = "windows"}}
+			{{- else if eq .OS "linux" -}}
+			{{$osStr = "linux"}}
+			{{- else if eq .OS "darwin" -}}
+			{{$osStr = "darwin"}}
+			{{- end -}}
+
+			{{$archiveStr := ""}}
+			{{- if HasPrefix .OS "ming" -}}
+			{{$archiveStr = ".zip"}}
+			{{- else -}}
+			{{$archiveStr = ".tar.gz"}}
+			{{- end -}}
+
+			https://github.com/{{.Owner}}/{{.Name}}/releases/download/v{{.Version}}/{{.Name}}_{{.Version}}_{{$osStr}}_amd64{{$archiveStr}}`,
```

Comment:
Could you try changing to BinaryTemplate and removing `https://github.com/{{.Owner}}/{{.Name}}/releases/download/` for a new feature I've added? And test removing the version.